project/iterative_sorting.py

```diff
@@ -2,47 +2,30 @@
 def selection_sort( arr ):
     # loop through n-1 elements
     for i in range(0, len(arr) - 1):
-        cur_index = i
-        smallest_index = cur_index
+        minimum = i
         # TO-DO: find next smallest element
         # (hint, can do in 3 loc) 
-        for j in range(i + 1, len(arr)):
-            if arr[j] < arr[cur_index]:
-                cur_index = j
-            if cur_index != i:
-                arr[i], arr[cur_index] = arr[cur_index], arr[i]
-
-
-
+        for j in range(minimum, len(arr)):
+            if arr[j] < arr[minimum]:
+                minimum = j
         # TO-DO: swap
-
-
-
-
+        arr[i], arr[minimum] = arr[minimum], arr[i]
     return arr
 
 
 # TO-DO: implement the Insertion Sort function below
 def insertion_sort( arr ):
-    for i in range(1, len(arr)):
-        current_element = arr[1]
-        j = i -1
-        while j >= 0 and current_element < arr[j]
-            arr[j + 1] = arr[j]
-            j -= 1
-        arr[j + 1] = current_element
+  for i in range(1, len(arr)):
+    j = i - 1
+    while arr[j] > arr[j + 1] and j >= 0:
+      arr[j], arr[j + 1] = arr[j + 1], arr[j]
```

Comment:
interesting! this is a different solution than what we worked with in class (but obviously it also works great!) great work!